src/connectors/playirish.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+Connector.playerSelector = '.qt-musicplayer';
+
+Connector.artistTrackSelector = '#streamCurrentlyPlaying';
+
+Connector.getArtistTrack = () => {
+	let text = $(Connector.artistTrackSelector).text().substring(16);
+	let m = text.match(/ - /g);
+	if (m && m.length === 2) {
```

Comment:
Have you any example where are two dashes in the artistTrackString?

Reply:
This check was actually not needed.